manifests/j/JetBrains/Aqua/EAP/223.7571.188/JetBrains.Aqua.EAP.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created with YamlCreate.ps1 v2.2.1 $debug=QUSU.CRLF.5-1-22621-963.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: JetBrains.Aqua.EAP
+PackageVersion: 223.7571.188
+InstallerType: nullsoft
+UpgradeBehavior: uninstallPrevious
+Installers:
+- Architecture: x64
+  InstallerUrl: https://download-cdn.jetbrains.com/aqua/aqua-223.7571.188.exe?_ga=2.67872661.584585179.1674607922-936843137.1674607922&_gl=1*tz19fw*_ga*OTM2ODQzMTM3LjE2NzQ2MDc5MjI.*_ga_9J976DJZ68*MTY3NDYwNzkyMS4xLjEuMTY3NDYwOTg4My41OC4wLjA.
+  InstallerSha256: CBCD02A3F26C797692E91FB0D5495D597B3D7C2841BCAA0E438966B3BD315C41
```

```suggestion
  InstallerSha256: CBCD02A3F26C797692E91FB0D5495D597B3D7C2841BCAA0E438966B3BD315C41
  ProductCode: Aqua 223.7571.188
```
For better upgrade matching